test/encapsulated-error-handler.test.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
+
+const { test } = require('tap')
+const Fastify = require('..')
+
+test('encapuslates an error handler', async t => {
+  t.plan(3)
+
+  const fastify = Fastify()
+  fastify.register(async function (fastify) {
+    fastify.setErrorHandler(async function a (err) {
+      t.equal(err.message, 'kaboom')
+      throw new Error('catched')
```

Comment:
It should be "caught".

Reply:
@mcollina You haven't changed all occurances. There are still two incorrect verbs used.